src/app/contextBroker/contextBroker.cpp

```diff
@@ -971,22 +968,16 @@ int main(int argC, char* argV[])
   //
   // disable file logging if the corresponding option is set. 
   //
-  if (disableFileLog)
+  if (paIsSet(argC, argV, (PaArgument *) paArgs, "-disableFileLog"))
   {
-    paConfig("log to file",                   (void*) false);
+    paConfig("log to file", (void*) false);
   } 
   else
   {
-    paConfig("log to file",                   (void*) true);
+    paConfig("log to file", (void*) true);
   }
-  
-  // setup logging after the env vars evaluated and log config is set.
-  if(paLogSetup() == -1) {
-    printf("Was not able to set up the logging system.");
-    exit(1);
-  }
```

Comment:
This is a bit weird ...
Looks like new code (as it doesn't follow the style guide) but ... it's removed ...
Would be nice to understand what happened here.

Reply:
This was from my first approach, when I extracted the log setup from the paParse method and called it explicitly after the env vars where evaluated. Since paIsSet now takes env-vars into account, it not longer required.